thunder/benchmarks/__init__.py

```diff
@@ -11,6 +11,7 @@
 from numbers import Number
 from typing import Any
 from contextlib import contextmanager
+from transformers import AutoConfig
```

Comment:
This file doesn't depend on the transformers project yet. Could you please move this import to the place where it's used, so that `thunder.benchmarks` is still importable even when `transformers` is not installed.